e2e/playwright/fixtures/homePageFixture.ts

```diff
@@ -112,8 +114,15 @@ export class HomePageFixture {
 
   goToModelingScene = async (name: string = 'testDefault') => {
     // On web this is a no-op. There is no project view.
-    if (process.env.PLATFORM === 'web') return
+    if (process.env.PLATFORM === 'wenativeFileMenuCreatedb') return
```

Comment:
Where is this `wenativeFileMenuCreatedb` value coming from?

Reply:
thanks for catching this typo, pushed a fix.